docs/design/INTEGRATE-ADMINAPI.V1.md

```diff
@@ -0,0 +1,488 @@
+
+    if (path.StartsWith("/v1/", StringComparison.InvariantCultureIgnoreCase))
+        return true;   
+    return false;
+}
+
+public async Task InvokeAsync(HttpContext context, RequestDelegate next)
+{
+ 
+        // Check if this is a V1 endpoint
+        if (IsV1Endpoint(context))
+        {
+            // For V1 endpoints, skip multi-tenancy validation entirely
+            await next.Invoke(context);
+            return;
+        }
+
+     if (multiTenancyEnabled)
+     {
+     }
+}
+
+```
+
+## Phase 7: Docker Setup
+
+**Objective**: Update Docker files to include Admin API V1 specific changes and
+support separate V1/V2 database infrastructure.
+
+**Tasks**:
+
+* Update Build Stage for V1 Project Integration
+**Files to modify**: dev.mssql.Dockerfile and dev.pgsql.Dockerfile
+  
+```docker
+# Add after existing project copies
+
+COPY --from=assets ./Application/NuGet.Config EdFi.Ods.AdminApi.V1/
+COPY --from=assets ./Application/EdFi.Ods.AdminApi.V1 EdFi.Ods.AdminApi.V1/
+```
+
+* **Docker Compose Considerations**: Add V1 specific environment variables if any.
+  
+ ```yml
+
+  adminapi:  
+    environment:
+      # Existing V2 configuration
+      ConnectionStrings__v2__EdFi_Admin: "host=pb-admin;port=${PGBOUNCER_LISTEN_PORT:-6432};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=EdFi_Admin;pooling=false"
+      ConnectionStrings__v2__EdFi_Security: "host=pb-admin;port=${PGBOUNCER_LISTEN_PORT:-6432};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=EdFi_Security;pooling=false"
+      
+      # New V1-specific database connections
+      ConnectionStrings__v1__EdFi_Admin: "host=pb-admin-v1;port=${PGBOUNCER_LISTEN_PORT:-6432};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=EdFi_Admin;pooling=false"
+      ConnectionStrings__v1__EdFi_Security: "host=pb-admin-v1;port=${PGBOUNCER_LISTEN_PORT:-6432};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=EdFi_Security;pooling=false"
+
+
+      # Update on compose file to use the unified database container:
+
+    services:
+      # Unified database container for both V1 and V2
+    db-admin:
+        build:
+        context: ../../../
+        dockerfile: Docker/db.pgsql.admin.unified.Dockerfile
+        environment:
+        POSTGRES_USER: ${POSTGRES_USER}
+        POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+        POSTGRES_DB: postgres
+        volumes:
+        - vol-db-admin:/var/lib/postgresql/data
+        restart: always
+        container_name: ed-fi-db-admin-unified
+        healthcheck:
+        test: ["CMD-SHELL", "pg_isready -U ${POSTGRES_USER}"]
+        start_period: "60s"
+        retries: 3
+
+ ```
+
+* `db.pgsql.admin.unified.Dockerfile` update:
+  `edfialliance/ods-api-db-admin:7.3` will be used as base image which contains
+  EdFi_Admin and EdFi_Security databases with 7.x schema. V1-specific EdFi_Admin
+  and EdFi_Security database schema files (6.x) will be downloaded as Azure
```

Comment:
I don't think this will work. You can't use the 7.3 admin database as a base layer for 6.2. I think we'll need two different Dockerfiles, or we'll need a variable controlling the base image tag.